hubblestack/returners/splunk_nova_return.py

```diff
@@ -169,6 +169,11 @@ def _generate_event(args, cloud_details, custom_fields, check_type=None, data=No
             for key, value in data[args['check_id']].items():
                 if key not in ['tag']:
                     event[key] = value
+    if check_type == 'Failure':
+        if 'detail' in data:
+            detail = data['detail']
+            for key in detail.keys():
```

Comment:
`for key in detail` is probably fine. no reason to invoke `.keys()` that I can think of. Though It's roughly the same thing, so I nearly didn't mention it.

Reply:
I have pushed this change out.